README.md

```diff
@@ -16,6 +16,16 @@ _**Please read the full README before using this template.**_
 
 ## Usage
 
+### Deploying `CreatorTokenFactory` using deploy script
+
+1. Specify the RPC URL for the desired network in the `.env` file.
+2. Specify the private key of the deployer address in the `.env` file.
+3. Run the following command to deploy the CreatorTokenFactory contract:
+
+```bash
+forge script Deploy --sig "run(address)" <ShowtimeVerifier Contract Address> --rpc-url $RPC_URL_OF_CHOICE --private-key $PRIVATE_KEY --broadcast
```

Comment:
A couple of things:

* I think the flag is `--fork-url` not `--rpc-url`
* Providing `.env` file values on the CLI like this won't actually work. It's not ideal but for what we're doing here I think it's fine to expect the user to provide them with their private key.

A more robust version of this would use cheatcodes to pull the key from the env var inside the script, but it's probably overkill for what we need here anyway.

Reply:
Why wouldn't it work? I did `source .env` and tried it on a counter contract with `rpc-url` flag, and it was deployed successfully.

Reply:
Made the changes here: [8daaab0](https://github.com/showtime-xyz/creator-tokens/pull/40/commits/8daaab026d8cbaafa76259cc2f4686d267a5717e)